From: Ondrej Jirman <megous@megous.com>
Date: Mon, 18 Oct 2021 17:41:25 +0200
Subject: arm64: dts: sun50i-a64-pinephone: Add supply for i2c bus to anx7688

The bus power needs to be enabled for anx7688 to be able to communicate
with the master.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index 3ca61d4..f0673d5 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -364,6 +364,8 @@ anx7688: hdmi-bridge@28 {
 		dvdd10-supply = <&reg_anx1v0>;
 		avdd18-supply = <&reg_ldo_io1>;
 		dvdd18-supply = <&reg_ldo_io1>;
+		avdd33-supply = <&reg_dldo1>;
+		i2c-supply = <&reg_ldo_io0>;
 		vconn-supply = <&reg_vconn5v0>;
 		hdmi_vt-supply = <&reg_dldo1>;
 
